src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -46,16 +30,43 @@
             }
         };
 
+        /**
+         * Represents the input field of the autocomplete.
+         *
+         * Events on the input field like onChange are defined here.
+         *
+         * @@type {{name: string, element: undefined, spinnerElement: undefined, spinnerTemplate: string, showSpinner: showSpinner, hideSpinner: hideSpinner, onChange: onChange, onNonInputKeys: onNonInputKeys, init: init}}
+         */
         var input = {
             name: "autocomplete-input",
             element: undefined,
+            spinnerElement: undefined,
+            spinnerTemplate: '<span class="autocomplete-spinner hide"><i class="fa fa-spinner fa-spin"></i></span>',
+
+            /**
+             * Shows a small spinner in the input field whenever the service loads data.
+             */
+            showSpinner: function () {
+                if (input.spinnerElement) {
```

Comment:
Would be a lot safer to check if it is a jQuery object (will actually have method removeClass):
`input.spinnerElement instanceof jQuery`